types/EligibilityCheck.ts

```diff
@@ -0,0 +1,57 @@
+import * as t from "io-ts";
```

Comment:
we need some unit tests that check if we can map this type to the generated one (API)